core/templates/services/UpgradedServices.ts

```diff
@@ -346,6 +346,9 @@ import { QuestionBackendApiService } from
   'domain/question/question-backend-api.service.ts';
 import { RatingComputationService } from
   'components/ratings/rating-computation/rating-computation.service';
+import { RatioExpressionInputRulesService } from
+  // eslint-disable-next-line max-len
+  'interactions/RatioExpressionInput/directives/ratio-expression-input-rules.service';
```

Comment:
I am sorry I can't allow new additions to this file unless absolutely necessary. @kevintab95 whats the timeline for this PR. It is marked for the current release.

Reply:
@srijanreddy98 please see EricZLou's analysis [here](https://github.com/oppia/oppia/issues/11920#issuecomment-783610564). He thinks that this service should have been added to `UpgradedServices.ts` earlier but was missed. Do you disagree with that analysis?

Reply:
Regarding timeline, this may fix a blocking bug for the February release

Reply:
@srijanreddy98 we need this fix to be merged ASAP to unblock the release. Please prioritise.

Reply:
@U8NWXD I think these changes may have not solved the problem. Just mentioning the service in UGS doesn't do anything. There is an array called servicesToProvide in App.ts. If the service is present there, then only adding to UGS makes sense.

Reply:
@srijanreddy98 I see what you mean. Are there any other ways for `InteractionRulesRegistryService` to pick up changes in `UpgradedServices`? 

See comments [here](https://github.com/oppia/oppia/issues/11920#issuecomment-784380299) for reproduction steps and screen recordings.